qunicorn_core/core/pilotmanager/qmware_pilot.py

```diff
@@ -140,6 +140,9 @@ def _get_job_results(self, qunicorn_job_id: int) -> None:  # noqa: C901
         if qunicorn_job is None:
             raise ValueError("Unknown Qunicorn job id {qunicorn_job_id}!")
 
+        jobs_to_save = []
+        results_to_save = []
```

Comment:
this could be combined into a list that saves `(PilotJob, list[PilotJobResult])` tuples but is probably ok as is